extension/src/shared/utils/files.ts

```diff
@@ -0,0 +1,23 @@
+export const getExtension = (filename?: string) => {
```

Comment:
Could you port test as well?
https://github.com/eukarya-inc/reearth-plateauview/blob/0080b7ddc6369447329db217653ac4f6ec461d56/plugin/web/extensions/sidebar/utils/file.test.ts